pkg/asset/config/appliance_config.go

```diff
@@ -91,6 +94,17 @@ sshKey: ssh-key
 # Password of user 'core' for connecting from console
 # [Optional]
 userCorePass: user-core-pass
+# [Optional]
+imageRegistry:
+  # The image registry container image to use.
+  # Default: docker.io/library/registry:2
+  # Alternative: quay.io/libpod/registry:2.8
```

Comment:
DId full cluster installation work fine with this image?

Reply:
waiting for results. still installing

Reply:
```
oc get co
NAME                                       VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
authentication                             4.12.22   True        False         False      2m29s
baremetal                                  4.12.22   True        False         False      7m24s
cloud-controller-manager                   4.12.22   True        False         False      7m17s
cloud-credential                           4.12.22   True        False         False      7m30s
cluster-autoscaler                         4.12.22   True        False         False      7m2s
config-operator                            4.12.22   True        False         False      15m
console                                    4.12.22   True        False         False      2m9s
control-plane-machine-set                  4.12.22   True        False         False      7m37s
csi-snapshot-controller                    4.12.22   True        False         False      7m15s
dns                                        4.12.22   True        False         False      7m6s
etcd                                       4.12.22   True        False         False      13m
image-registry                             4.12.22   True        False         False      2m30s
ingress                                    4.12.22   True        False         False      6m54s
insights                                   4.12.22   True        False         False      7m16s
kube-apiserver                             4.12.22   True        False         False      4m21s
kube-controller-manager                    4.12.22   True        False         True       4m19s   GarbageCollectorDegraded: error fetching rules: Get "https://thanos-querier.openshift-monitoring.svc:9091/api/v1/rules": dial tcp 172.30.175.201:9091: connect: connection refused
kube-scheduler                             4.12.22   True        False         False      5m8s
kube-storage-version-migrator              4.12.22   True        False         False      9m30s
machine-api                                4.12.22   True        False         False      7m7s
machine-approver                           4.12.22   True        False         False      7m40s
machine-config                             4.12.22   True        False         False      6m10s
marketplace                                4.12.22   True        False         False      7m20s
monitoring                                 4.12.22   True        False         False      98s
network                                    4.12.22   True        False         False      15m
node-tuning                                4.12.22   True        False         False      7m4s
openshift-apiserver                        4.12.22   True        False         False      2m49s
openshift-controller-manager               4.12.22   True        False         False      2m20s
openshift-samples                          4.12.22   True        False         False      3m58s
operator-lifecycle-manager                 4.12.22   True        False         False      7m1s
operator-lifecycle-manager-catalog         4.12.22   True        False         False      7m7s
operator-lifecycle-manager-packageserver   4.12.22   True        False         False      4m34s
service-ca                                 4.12.22   True        False         False      15m
storage                                    4.12.22   True        False         False      7m12s
```